jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/form/CheckboxAndRadioButtonCustomDisabledTests.java

```diff
@@ -18,12 +19,13 @@
 public class CheckboxAndRadioButtonCustomDisabledTests extends TestsInit {
 
     private String label1 = "Toggle this custom radio";
+    WebList radioBtnList;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/jdi-testing/jdi-light/pullRequest?prid=4482631)